os/x11/xinput.h

```diff
@@ -159,7 +159,7 @@ class XInput {
 
     ASSERT(XSelectExtensionEvent);
     XSelectExtensionEvent(display, window,
-                          &m_eventClasses[0],
+                          m_eventClasses->data(),
```

Comment:
m_eventClasses is not a pointer but a std::vector, so we have to access data() member with the . operator:
```suggestion
                          m_eventClasses.data(),
```